azext_edge/edge/providers/orchestration/upgrade.py

```diff
@@ -105,7 +109,8 @@ def do_work(self, confirm_yes: Optional[bool] = None):
 
```

Comment:
Is `--no-progress` actually adhered to? Various prints to stdout

Reply:
oh I thought delete also still printed - gonna make sure they aren't printed then

Reply:
delete should not print when --no-progress is used, with the exception of the confirm prompt which can be hidden with -y.

Reply:
![image](https://github.com/user-attachments/assets/9a77621b-dfc1-4f6a-b3df-2bbb7b20226a)
fixed